packages/js/examples/react-audio/stories/components/WebDialer.jsx

```diff
@@ -165,6 +167,13 @@ const WebDialer = ({
     }
   };
 
+  const toggleDeaf = () => {
+    if (call) {
+      setIsToggle(!isToggle);
```

Comment:
You could do something similar to what I did here with held state: https://github.com/team-telnyx/webrtc/pull/91/files#diff-7fe522d30a42863e57a35b3c5f34faca480c6ba80bf3a33d8549e724625156f2R171

Although per my previous comment, I think you meant to do `call.isDeaf`

Reply:
Yes, you are right. I will change :-)